Readme update

rlondner opened the pull request:
Fixed a few typos and added some clarifications

rlondner committed:
Readme update

Various corrections

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PureStorage-OpenConnect/ansible-playbook-examples?pullRequest=2) <br/>All committers have signed the CLA.

rlondner committed:
2.6 scripts enhancements

Added Ubuntu support
Parameterized FA Url and API Tokens
Fixed typo in multihost_provisioning_cleanup.yaml

rlondner commented:
Added Ubuntu support in multihost_provisioning.yaml
Parameterized FA Url and API Tokens in multihost_provisioning.yaml and multihost_provisioning_cleanup.yaml
Fixed typo in multihost_provisioning_cleanup.yaml
